src/app/components/README.md

```diff
@@ -0,0 +1,3 @@
+# Components
+
+In general this should not be used. Components should be written, edited and released via [our component library Psammead](https://github.com/bbc/psammead).
```

Comment:
Can we give an example of where this should be used? Or are we trying to stop people putting components here at all?

Reply:
Done